### DISCUSSION TITLE
Documentation missing location of config files (and user)

### DISCUSSION THREAD

**Discussion by silverhook** [posativ/isso#787] [Q&A]
I’m unable to find in the documentation:
- how to select the normal user in question – does it have to be the same user as the one that the webserver is running under, or would any user do (an example would be nice);
- which directory should the configuration files be saved to; and (often even)
- what the name of the configuration files should be.

**Comment by posativ** [comment]
It depends. I usually run all web applications written in Python from a single user called `uwsgi` because I use [uWSGI](https://github.com/unbit/uwsgi) as application server. You may run Isso from a dedicated user, the usual communication between application and webserver is done via TCP (on localhost).

The configuration file needs to be readable by the user that als runs Isso. There's no way to drop privileges natively, but certain application servers (I think Gunicorn and uWSGI) can be configured to do so.

The name of the configuration file can be anything, there's not default location where Isso would look for a configuration. It is either passed on the command line or using environment variables (described in [Deployment](http://posativ.org/isso/docs/extras/deployment/)).

---

What do you want to change for the documentation? I don't think the documentation should enforce a certain user or even a certain web server. It is a WSGI application and thus is very flexible in its usage (which is hopefully good documented in the deployment document).

**Comment by silverhook** [comment]
Excuse my noobieness …if I’m running this on Nginx, is just enabling its [uwsgi_module](http://nginx.org/en/docs/http/ngx_http_uwsgi_module.html) enough …or even just depending on the builtin server? I’m confused in that part.
